docs/node-operators/get-started/setup-datanode.md

```diff
@@ -73,9 +73,12 @@ Where:
 - `database_password` is the password you want to use to connect to the database.
 - `database_name` is the name of the database you want to use for storing the data.
 - `localdb_port` is the port you want to use to connect to the database on your local machine. (5432 is the default port for Postgresql database server and may not be available if you already have a postgresql database server running on your machine and want to use Docker for testing).
-- `/host_path/to/snapshotsCopyTo` is the path on your host machine where you want to store the snapshots that are generated by data node. Snapshots allow you to restore the database to a previous state.
```

Comment:
Sort of gives the impression that this is where we store the network history, when really they are temp working used when creating/loading history segments, however, I think the change Phil is making will remove the need for these paths, so perhaps not worth updating this for now.